txp/Pos/Txp/Core/Types.hs

```diff
@@ -114,6 +115,12 @@ instance NFData TxInWitness
 -- is provided for each input.
 type TxWitness = Vector TxInWitness
 
+instance Buildable TxWitness where
+  build v = foldr (\txWit recV -> B.fromText "{ " `mappend`
```

Comment:
Very very minor detail: Use `<>` instead of `mappend` (when used in infix form).